packages/main/src/Table.ts

```diff
@@ -512,7 +498,7 @@ class Table extends UI5Element {
 	}
 
 	get _tableOverflowX() {
-		return (this.overflowMode === TableOverflowMode.Popin) ? "hidden" : "auto";
+		return (this.overflowMode === TableOverflowMode.Popin) ? "" : "auto";
```

Comment:
default is visible are you sure that is really what we want for the popin. what is the motivation behind it?

Reply:
changed it to clip. hidden causes problems with the sticky header (I guess because hidden is seens as an overflow and therefore requires the container to have a fixed height?), changed it to clip